docs/collisions.md

```diff
@@ -0,0 +1,564 @@
+<p>
+`sync`: permanent destination now matches the local source exactly: the same files, the same content.
+   </td>
+  </tr>
+  <tr>
+   <td><strong>E</strong>
+   </td>
+   <td>Permanent (empty)
+   </td>
+   <td>Local (empty)
+   </td>
+   <td>Nothing happens, they’re both empty
+   </td>
+   <td>No change
+   </td>
+  </tr>
+  <tr>
+   <td><strong>F</strong>
+   </td>
+   <td>Permanent (empty)
+   </td>
+   <td>Local (populated)
+   </td>
+   <td>Depending on the command used in rclone (sync vs copy), this will either:
+<p>
+`copy`: Do nothing.
+<p>
+`sync`: Delete all content of the local directory.
+   </td>
+   <td>`copy`: no change
+<p>
+`sync`: local destination is now empty.
+   </td>
+  </tr>
+  <tr>
+   <td><strong>G</strong>
+   </td>
+   <td>Permanent (populated)
+   </td>
+   <td>Local (empty)
+   </td>
+   <td>Normal download of all files and folders.
+   </td>
+   <td>Local now has copies of each file and folder containing the same content as Permanent.
+<p>
+<em>Note: the names of files and folders are determined by the “dedupe” algorithm described earlier in this document.</em>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>H</strong>
+   </td>
+   <td>Permanent (populated)
+   </td>
+   <td>Local (populated)
+   </td>
+   <td>Depending on the command used in rclone (sync vs copy), this will either:
+<p>
+`copy`: Download content for each file that has a difference (determined by rclone)
+<p>
+`sync`: Download content for each file with a difference, delete any files or folders that don’t exist on the Permanent source.
+<p>
+<em>Note: that file destination paths are deterministic / all “duplicate” file names on the Permanent end had already been resolved by the dedupe naming algorithm described earlier in this document.</em>
+   </td>
+   <td>`copy`: all files that exist on Permanent now exist on the local filesystem, and the content matches exactly.  However, files on the local filesystem that don’t exist on Permanent have not been modified.
+<p>
+`sync`: local filesystem now matches the Permanent source exactly: the same files, the same content.
+   </td>
+  </tr>
+</table>
+
+
+## Deletion Risks
+
+We want to make sure that users understand the difference between a **sync **and a **copy. ** Those two commands mean different things in rclone:
+
+
+* **“sync”** is more destructive because it can delete files.  For example syncing an empty local folder with a populated Permanent archive would lead to the complete deletion of the archive which may not be what the user expected. Hence the buttons or whatever UI is exposed to the user for rclone interaction must provide the appropriate warning.
+* **“copy” **is less destructive because it does not delete files, it either creates or overwrites.  It can still cause data modification in the case of an overwrite. For example if a file exists both locally and on Permanent but with different content.  After the copy both files would contain the same content.
```

Comment:
typo here (space after "copy")

Reply:
Thanks Dan, there were a couple of those and I have fixed them.